### ISSUE TITLE
Exception in Pod::ProjectDocs::BUILDARGS: Attribute (except) does not pass the type constraint

### ISSUE THREAD

**Issue by ehuelsmann** [mgruner/p5-pod-projectdocs#5]
We're using Pod::ProjectDocs to generate [LedgerSMB](https://github.com/ledgersmb/LedgerSMB) documentation (at least every now and then, but plans are to do it on every release).

Unfortunately, I'm experiencing problems with the `except` attribute type definition in Pod::ProjectDocs::Config. Here is our [documentation generator script](https://github.com/ledgersmb/LedgerSMB/blob/6a441c7c033229157e1e14e52db1b996fd70e144/utils/pod2projdocs.pl). It sets the except attribute to an array of regex references, just as per the documentation.

When running our `make` command, the docs fail to build with the following output:

```plain
$ make pod
rm -rf UI/pod
mkdir UI/pod
utils/pod2projdocs.pl 2>&1 pod2projdocs.log
Attribute (except) does not pass the type constraint because: Validation failed for 'ArrayRef[Str]' with value ARRAY(0xa1747f4f48) at /usr/lib/x86_64-linux-gnu/perl5/5.24/Moose/Object.pm line 24
	Moose::Object::new('Pod::ProjectDocs::Config', 'title', 'LedgerSMB Documentation', 'forcegen', 0, 'outroot', '/srv/ledgersmb/UI/pod', 'libroot', 'ARRAY(0xa174812588)', 'desc', 'manuals and libraries', 'except', 'ARRAY(0xa1747f4f48)', 'lang', 'en') called at /usr/local/share/perl/5.24.1/Pod/ProjectDocs.pm line 62
	Pod::ProjectDocs::BUILDARGS('Pod::ProjectDocs', 'outroot', 'UI/pod', 'except', 'ARRAY(0xa1747f4f48)', 'libroot', './', 'title', 'LedgerSMB Documentation') called at /usr/lib/x86_64-linux-gnu/perl5/5.24/Moose/Object.pm line 22
	Moose::Object::new('Pod::ProjectDocs', 'outroot', 'UI/pod', 'except', 'ARRAY(0xa1747f4f48)', 'libroot', './', 'title', 'LedgerSMB Documentation') called at utils/pod2projdocs.pl line 6
Makefile:313: recipe for target 'pod' failed
make: *** [pod] Error 2
```

The above is based on the the latest installed from CPAN today:

```plain
$ cpanm Pod::ProjectDocs
--> Working on Pod::ProjectDocs
Fetching http://www.cpan.org/authors/id/M/MG/MGRUNER/Pod-ProjectDocs-0.49.tar.gz ... OK
Configuring Pod-ProjectDocs-0.49 ... OK
==> Found dependencies: Test::Pod
--> Working on Test::Pod
Fetching http://www.cpan.org/authors/id/E/ET/ETHER/Test-Pod-1.51.tar.gz ... OK
Configuring Test-Pod-1.51 ... OK
Building and testing Test-Pod-1.51 ... OK
Successfully installed Test-Pod-1.51
Building and testing Pod-ProjectDocs-0.49 ... OK
Successfully installed Pod-ProjectDocs-0.49
2 distributions installed
```

**Comment by mgruner** [comment]
Fixed by applying pull request #6.

**Comment by mgruner** [comment]
New release 0.51 is uploaded.
